www/form/inner.js

```diff
@@ -5099,9 +5136,30 @@ define([
                 editableStr
             ]);
 
+            var toggleOffclass = 'ontouchstart' in window ? 'toggle-active' : 'toggle-inactive'; 
+            var toggleOnclass = 'ontouchstart' in window ? 'toggle-inactive' : 'toggle-active'; 
+            var toggleDragOff = h(`button#toggle-drag-off.cp-form-view-drag.${toggleOffclass}.fa.fa-arrows`, {'aria-hidden': true, 'title': Messages.toggleArrows});
+            var toggleDragOn = h(`button#toggle-drag-on.cp-form-view-drag.${toggleOnclass}.fa.fa-hand-o-up`, {'aria-hidden': true, 'title': Messages.toggleDrag});
+            $(toggleDragOff).click(function() {
+                $(toggleDragOff).attr('class').indexOf('toggle-inactive') !== -1 ? $(toggleDragOff).toggleClass('toggle-inactive').toggleClass('toggle-active') && $(toggleDragOn).toggleClass('toggle-active').toggleClass('toggle-inactive') : undefined;
+                updateForm(framework, content, true);
+                APP.mainSortable.options.disabled =  true;
+            });
+            $(toggleDragOn).click(function() {
+                $(toggleDragOn).attr('class').indexOf('toggle-inactive') !== -1 ? $(toggleDragOn).toggleClass('toggle-inactive').toggleClass('toggle-active') && $(toggleDragOff).toggleClass('toggle-active').toggleClass('toggle-inactive') : undefined;
+                updateForm(framework, content, true);
+                APP.mainSortable.options.disabled =  false; 
+            });
+
+            var drag = h('div', [
+                toggleDragOff,
+                toggleDragOn
+            ]);
+
             return [
                 preview,
                 previewSettings,
+                drag,
```

Comment:
A small issue about the style on mobile, this could also probably be fixed in the .less files directly:
![image](https://github.com/user-attachments/assets/2e7daad6-382b-43e0-ab3b-f0fdddc67e2c)